scipy/integrate/tests/test_quadpack.py

```diff
@@ -259,7 +260,74 @@ def test_double_integral3(self):
         def func(x0, x1):
             return x0 + x1 + 1 + 2
         assert_quad(dblquad(func, 1, 2, 1, 2),6.)
-        
+
+    @pytest.mark.parametrize(
+        "x_lower, x_upper, y_lower, y_upper, expected",
+        [
+            # Multiple integration of a function in n = 2 variables: f(x, y, z)
```

Comment:
I'm not sure if this line needs to be repeated for each test case. Ditto for tplquad.

Reply:
I would generally agree. I thought I'd put these information comments in just to make it clarify what each parameterisation is covering, and also to break it up a bit. If we'd prefer them to be removed, that's not a problem at all! Just let me know and I can get onto it.

Reply:
I'm not strongly opinionated either way. If you feel like it's worth doing, go ahead. I'm going to leave this PR open for another couple of days, just in case anyone else has any opinion.

Reply:
Looks fine to me. As does this PR as a whole. I'll leave it to @Kai-Striega to merge